pkg/task/display.go

```diff
@@ -0,0 +1,107 @@
+package task
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+	"github.com/turbot/go-kit/files"
+	"github.com/turbot/steampipe/pkg/filepaths"
+	"github.com/turbot/steampipe/pkg/utils"
+)
+
+const (
+	NotificationsStructVersion = 20221121
+)
+
+type Notifications struct {
+	StructVersion           uint32   `json:"struct_version"`
+	CLINotificationsLines   []string `json:"cli_notifications"`
+	PluginNotificationLines []string `json:"plugin_notifications"`
+}
+
+func (r *Runner) saveNotifications(cliNotificationsLines, pluginNotificationLines []string) error {
+	utils.LogTime("Runner.saveNotifications start")
+	defer utils.LogTime("Runner.saveNotifications end")
+
+	if len(cliNotificationsLines)+len(pluginNotificationLines) == 0 {
+		// nothing to save
+		return nil
+	}
+
+	notifs := &Notifications{
+		StructVersion:           NotificationsStructVersion,
+		CLINotificationsLines:   cliNotificationsLines,
+		PluginNotificationLines: pluginNotificationLines,
+	}
+	if files.FileExists(filepaths.NotificationsFilePath()) {
```

Comment:
Not sure if this is necessary!

Reply:
well - can you check whether it is and remove it not